app/assets/stylesheets/application.scss

```diff
@@ -55,6 +55,19 @@ p {
   margin: 5px 0;
 }
 
+.colon-position-support {
```

Comment:
I wonder if we could rename this? It doesn't immediately make sense to me what this is styling.

Reply:
I leave this for Dushan 😀

Reply:
renamed this in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1018/commits/483c229410cbc6c7ac09da3b6f0837abf2621367 let me know what you think 😄